icons/ifixit.svg

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg viewBox="0 0 48 48" xmlns="http://www.w3.org/2000/svg">
```

Comment:
New icons should go in the `other` folder so my scripts can process them before a release.

Reply:
moved